python/graphstorm/config/argument.py

```diff
@@ -1650,7 +1650,7 @@ def sparse_optimizer_lr(self): # pylint: disable=invalid-name
 
     @property
     def use_node_embeddings(self):
-        """ Whether to use extra learnable node embeddings
+        """ Whether to use extra learnable embeddings for nodes. Default is False.
```

Comment:
Can we take the chance to add a more descriptive alias for this? We've gotten feedback that the name is not very intuitive. Something like `create_learnable_embeddings`?

And let's say "create" in the docstring instead of "use", use implies we will use something that's already there (causing confusion with node features), here we create the embeddings during training.

Reply:
Good call. Let's go over all doc string to let them more intuitive.